test/support/admin_edition_controller_test_helpers.rb

```diff
@@ -1609,10 +1613,10 @@ def assert_data_attributes_for_ministers(element:, track_label:)
   end
 
   def assert_data_attributes_for_worldwide_organisations(element:, track_label:)
-    assert_equal "Worldwide organisations…", element["data-placeholder"]
-    assert_equal "track-select-click", element["data-module"]
-    assert_equal "worldwideOrganisationSelection", element["data-track-category"]
-    assert_equal track_label, element["data-track-label"]
+    # TODO: Add tracking back in. This is covered in this Trello card https://trello.com/c/eKGeFCQu/975-add-tracking-in-for-associations-on-the-edit-page
+    # assert_equal "track-select-click", element["data-module"]
+    # assert_equal "worldwideOrganisationSelection", element["data-track-category"]
+    # assert_equal track_label, element["data-track-label"]
```

Comment:
It doesn't feel great to be merging this in. But I'm not sure what to suggest in its place.

Does this mean the Accessible Autocomplete component doesn't support the tracking that this page needs? Should we consider this a blocker?

Reply:
Yea for all the associations the autocompletes aren't tracked. Dilwoar mentioned there's a module that'll need to be rewritten. There's a card tracking places it needs to be added here  https://trello.com/c/eKGeFCQu/975-add-tracking-in-for-associations-on-the-edit-page